### PR TITLE
[Migrate Simple Payments] After a delay show bottom sheet

### LINKED ISSUE #11201: [Migrate Simple Payments] After a delay show bottom sheet
uycU7BYzmc3vLKX0cuHLNk-fi-60_42572

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #11201

### Description

The PR adds a bottom sheet as per design. The bottom doesn't do anything yet

uycU7BYzmc3vLKX0cuHLNk-fi-60_42572

### Testing instructions

* More -> Payments -> Collect Payment
* Notice the new bottom sheet with an explanation

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4923871/8e6ac398-f2c3-4419-9deb-8344740a9d53

image

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/204">18.1</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by backwardstruck** [comment]
I see the new bottom sheet:

Screenshot_20240403_142127

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt [hunk 1]
```diff
@@ -206,6 +206,7 @@ class OrderCreateEditViewModel @Inject constructor(
     companion object {
         val EMPTY_BIG_DECIMAL = -Double.MAX_VALUE.toBigDecimal()
         const val MAX_PRODUCT_QUANTITY = 100_000
+        const val DELAY_BEFORE_SHOWING_SIMPLE_PAYMENTS_MIGRATION_BOTTOM_SHEET = 500L
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt:209]: This value seems fine.

**Review by backwardstruck — APPROVED** [review]

**Comment by backwardstruck** [comment]
I'm not sure why the tests aren't passing.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt:209]: That numbers comes from Joe =)

**Pull request merged**